marketpulse/main/templates/media_new.html

```diff
@@ -57,6 +57,11 @@ <h2 class="title">New Photo</h2>
             <div>
               <a href="#" class="btn btn-default btn-block" id="pickimage" data-role="button">Pick Image</a>
             </div>
+            <div class="license-text">
+              all photo contributions are available under a
```

Comment:
I prefer to have more or less the same text we have on the footer. Not all people are aware of CC initials.